src/content/docs/en/installation/rpi.mdx

```diff
@@ -33,11 +33,18 @@ The best experience will be with a SATA SSD connected over an adapter. A USB 3.0
   it on another device.
 </Alert>
 
+First, you will need to decompress it.
+
+- On Linux, run `unzstd *FILENAME.img.zst*`, you may need to install the `zstd` package
+- On Windows 11 or ChromeOS, zstd is natively supported. Decompress like you would a Zip file.
+- On Windows 10, use 7zip
+- On macOS, install `zstd` from Homebrew (`brew install zstd`), then `unzstd *FILENAME.img.zst*`.
+
 Grab a USB stick! Use a flasher program to flash Ultramarine to your storage device. [balenaEtcher](https://www.balena.io/etcher/) is an easy cross-platform choice.
```

Comment:
USB may be too specific here

Reply:
Can fix that then lol

Reply:
Actually, storage devices are covered earlier, thoughts on just removing 'Grab a USB stick!'?